src/onegov/org/models/organisation.py

```diff
@@ -179,6 +179,10 @@ class Organisation(Base, TimestampMixin):
     gever_password = meta_property()
     gever_endpoint = meta_property()
 
+    # data retention policy
+    auto_archive_timespan = meta_property()
+    auto_delete_timespan = meta_property()
```

Comment:
If you change these to have a default value then you don't need a custom `process_obj` method in the form.
```suggestion
    auto_archive_timespan = meta_property(default=0)
    auto_delete_timespan = meta_property(default=0)
```